deploy/templates/deployment.yaml

```diff
@@ -24,6 +24,24 @@ spec:
             requests:
               memory: "64Mi"
               cpu: "10m"
+          livenessProbe:
+            tcpSocket:
+              port: downloads
+            initialDelaySeconds: 15
```

Comment:
Do we need this long of a delay?

Reply:
Nope! I wasn't sure what was appropriate, but it's definitely a small deployment--I'll knock it down.